vars/buildDockerAndPublishImage.groovy

```diff
@@ -280,20 +284,38 @@ def call(String imageShortName, Map userConfig=[:]) {
           ]) {
             String release = ''
             if (isUnix()) {
-              final String releaseScript = '''
-                originUrlWithGit="$(git remote get-url origin)"
-                originUrl="${originUrlWithGit%.git}"
-                org="$(echo "${originUrl}" | cut -d'/' -f4)"
-                repository="$(echo "${originUrl}" | cut -d'/' -f5)"
-                releasesUrl="/repos/${org}/${repository}/releases"
-                releaseId="$(gh api "${releasesUrl}" | jq -e -r '[ .[] | select(.draft == true and .name == "next").id] | max | select(. != null)')"
-                if test "${releaseId}" -gt 0
-                then
-                  gh api -X PATCH -F draft=false -F name="${TAG_NAME}" -F tag_name="${TAG_NAME}" "${releasesUrl}/${releaseId}" > /dev/null
-                fi
-                echo "${releaseId}"
-              '''
-              release = sh(script: releaseScript, returnStdout: true)
+              final String makeShowContent = makecall('retrieve-make-show', imageName, operatingSystem, finalConfig.dockerBakeFile, finalConfig.dockerBakeTarget)
+              withEnv(["MAKE_SHOW_CONTENT=${makeShowContent}"]) {
+                final String releaseScript = '''
+                  originUrlWithGit="$(git remote get-url origin)"
+                  originUrl="${originUrlWithGit%.git}"
+                  org="$(echo "${originUrl}" | cut -d'/' -f4)"
+                  repository="$(echo "${originUrl}" | cut -d'/' -f5)"
+                  releasesUrl="/repos/${org}/${repository}/releases"
+                  releaseId="$(gh api "${releasesUrl}" | jq -e -r '[ .[] | select(.draft == true and .name == "next").id] | max | select(. != null)')"
+                  if test "${releaseId}" -gt 0
+                  then
+                    body="$(gh api "${releasesUrl}/${releaseId}" | jq -e -r '.body')"
+                    body+='
+
+## :memo: Release settings
+
+<details>
+
+```json
+'
+                    body+="${MAKE_SHOW_CONTENT}"
```

Comment:
```suggestion
```yaml
'
                    body+="$(echo ${MAKE_SHOW_CONTENT} | yq --prettyPrint"
```
Would output more readable YAML instead of JSON:
```yaml
target:
  default:
    context: /home/jenkins/workspace/bs_docker-confluence-data_0.1.35
    dockerfile: /home/jenkins/workspace/bs_docker-confluence-data_0.1.35/Dockerfile
    args:
      BUILD_DATE: "2024-02-09T15:35:16Z"
      GIT_COMMIT_REV: 12f63021
      GIT_SCM_URL: https://github.com/jenkins-infra/docker-confluence-data.git
    labels:
      org.label-schema.build-date: "2024-02-09T15:35:16Z"
      org.label-schema.url: '"https://github.com/jenkins-infra/docker-confluence-data.git"'
      org.label-schema.vcs-ref: 12f63021
      org.label-schema.vcs-url: https://github.com/jenkins-infra/docker-confluence-data.git
      org.opencontainers.image.created: "2024-02-09T15:35:16Z"
      org.opencontainers.image.revision: 12f63021
      org.opencontainers.image.source: https://github.com/jenkins-infra/docker-confluence-data.git
      org.opencontainers.image.url: '"https://github.com/jenkins-infra/docker-confluence-data.git"'
    tags:
      - docker.io/jenkinsciinfra/wiki:latest
      - docker.io/jenkinsciinfra/wiki:0.1.35
    platforms:
      - linux/amd64,linux/arm64
```